### PR TITLE
Add CVVResponseCode and constants to Transaction.

### PR DESCRIPTION
**Pull request by ccahoon**
I wasn't sure whether we wanted constants in this one-- I'm happy to pull them out, since they're not in the Java implementation. Also, unsure stylistically about the multiline comment in the constant file, but here's a draft. Feedback welcome.

### CODE REVIEW

**Comment by ccahoon** [comment]
#143

File: transaction_integration_test.go [hunk 1]
```diff
@@ -250,6 +250,42 @@ func TestTransactionCreateWhenGatewayRejectedFraud(t *testing.T) {
...
+			CVV:            "200", // Should cause CVV does not match response
+		},
+	})
+
+	if err.Error() != "Gateway Rejected: cvv" {
+		t.Fatal(err)
+	}
+
+	txnID := err.(*BraintreeError).Transaction.Id
+	txn, err := testGateway.Transaction().Find(txnID)
```
Comment (leighmcculloch) [inline transaction_integration_test.go]: Same comment as on #148. Do we need the find?

File: cvv_response_code.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	CVVMatches      CVVResponseCode = "M" // The CVV provided matches the information on file with the cardholder's bank.
+	CVVDoesNotMatch CVVResponseCode = "N" // The CVV provided does not match the information on file with the cardholder's bank.
+
+	// The card-issuing bank received the CVV but did not verify whether it was correct.
+	// This typically happens if the processor declines an authorization before the bank evaluates the CVV.
+	CVVNotVerified CVVResponseCode = "U"
+
+	CVVNotProvided              CVVResponseCode = "I" // No CVV was provided.
+	CVVIssuerDoesNotParticipate CVVResponseCode = "S" // The CVV was provided but the card-issuing bank does not participate in card verification.
+	CVVNotApplicable            CVVResponseCode = "A" // The CVV was provided but this type of transaction does not support card verification.
```
Comment (leighmcculloch) [inline cvv_response_code.go]: Same comment as on #148 that we should probably namespace these with the types name as the prefix. But interested to hear what you think as well as thoughts from @lionelbarrow.

Comment (ccahoon) [inline cvv_response_code.go]: I think it's a good idea. On it.

**Comment by ccahoon** [comment]
I think I've addressed all of your comments. Please take another look!

**Comment by leighmcculloch** [comment]
This looks great! Thanks @ccahoon!

**Pull request merged**